rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
Right. I wasn't really sure how to approach this though, since RecyclerView is a class in v7.widget and not its own package. I'd have to change the package hierarchy to also say something like `support.v7.recyclerview.widget.whatever` so it can find the test activity correctly, but that seems a little backward since it's in the widget package. Thoughts?